packages/perspective-viewer-d3fc/src/js/legend/scrollableLegend.js

```diff
@@ -33,6 +34,7 @@ export default fromLegend => {
     const render = selection => {
         renderControls(selection);
         renderLegend(selection);
+        cropCellContents(selection);

```

Comment:
Do we need this in legend and scrollableLegend?

Reply:
Nope, we don't.